### PR TITLE
[FLINK-14684] Add Pinterest to Chinese Powered By page

### PR DESCRIPTION
**Pull request by jiaweiwu-github**
Add Pinterest to Chinese Powered By page

related english PR: https://github.com/apache/flink-web/pull/281

### CODE REVIEW

File: poweredby.zh.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ Apache Flink 为全球许多公司和企业的关键业务提供支持。在这
     <img src="{{ site.baseurl }}/img/poweredby/ovh-logo.png" alt="OVH" /><br />
       OVH 使用 Flink 开发面向流的应用程序，比如实时商业智能系统或警报系统。<br><br><a href="https://www.ovh.com/fr/blog/handling-ovhs-alerts-with-apache-flink/" target='_blank'><small><span class="glyphicon glyphicon-new-window"></span></small> 详细了解 OVH 如何使用 Flink</a>
   </div>
+  <div class="powered-by-tile col-md-3 col-sm-4 col-xs-6">
+    <img src="{{ site.baseurl }}/img/poweredby/pinterest-logo.png" alt="Pinterest" /><br />
+      Pinterest 使用基于 Flink 的实时实验分析平台每天进行上千次的实验。 <br><br><a href="https://medium.com/pinterest-engineering/real-time-experiment-analytics-at-pinterest-using-apache-flink-841c8df98dc2" target='_blank'><small><span class="glyphicon glyphicon-new-window"></span></small> 详细了解 Pinterest 的实时实验分析平台</a>
```
Comment (hequn8128) [inline poweredby.zh.md]: 是不是要把平台去了，即：详细了解 Pinterest 的实时实验分析平台 => 阅读更多在 Pinterest 有关实时实验分析的信息 ?
According to the English description, it is the real-time experiment analytics, i.e., not real-time experiment analytics **platform**.

Comment (hequn8128) [inline poweredby.zh.md]: Flink => Apache Flink

Comment (jiaweiwu-github) [reply poweredby.zh.md]: Accepted. Thanks.

Comment (jiaweiwu-github) [reply poweredby.zh.md]: Accepted. Thanks!

**Comment by jiaweiwu-github** [comment]
@hequn8128 Thanks for the detailed review.
I have publish a new revision according to comments.

**Pull request closed without merging**